app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -97,12 +97,14 @@ public function register (Request $request)
     */
     public function confirmEmail ($token)
     {
-        // a dynamic / magic method:
-        $user = User::whereToken($token)->firstOrFail()->confirmEmail();
-
-        session()->flash('emailconfirmed', 'Your email has been confirmed. You may now Log in.');
-
-        return redirect('/');
+        // a dynamic / magic method
+        // bool
+        $verified = User::whereToken($token)->firstOrFail()->confirmEmail();
+        // \Log::info(['verified', $verified]);
+        $auth = Auth::check();
```

Comment:
You can set `$auth = false` here as the user will not be logged in as they need to verify their password to log in